src/bitgo.js

```diff
@@ -1089,7 +1094,12 @@ BitGo.prototype.authenticate = function(params, callback) {
     return this.reject('already logged in', callback);
   }
 
-  const request = this.post(this.url('/user/login'));
+  let request;
+  if (params.microservices && this._microservicesUrl) {
+    request = this.post(this.microservicesUrl('/api/v1/auth/session'));
```

Comment:
I'd rather you do something like this instead (to minimize the amount of duplicated code).

```js
const authUrl = this._microservicesURL ? this.microservicesUrl('/api/v1/auth/session') : this.url('/user/login');
const request = this.post(authUrl);
```

Reply:
will do.